src/pages/new-chat/new-chat.ts

```diff
@@ -9,22 +12,30 @@ import { User } from 'api/models/whatsapp-models';
   templateUrl: 'new-chat.html'
 })
 export class NewChatComponent implements OnInit {
-  users;
+  searchPattern = '';
```

Comment:
Read the other note about `ngModel` - I think this one should be a `Subject` that next each time the user changes the input, then use this `Subject` as part of combined stream, and then filter your data - I think that this way you might even can remove the `search` Pipe.